logstash.go

```diff
@@ -15,9 +16,10 @@ var LogLevels = map[string]log.Level{
 	"PANIC":   log.PanicLevel,
 }
 
-func Init(logLevel string, logFile string, env string, service string) bool {
+func Init(logLevel string, logFileName string, env string, service string) bool {
+
+	logFile, err := os.Create(logFileName)
```

Comment:
Are we sure we want to use `os.Create` ? it truncates the file it it already exists.
That could be an issue when restarting services. we can use [`os.OpenFile`](https://golang.org/pkg/os/#OpenFile) to append to an existing log file if it exists

Reply:
Good point, but the file location needs to be moved anyway, since that folder gets deleted on restart.